commands/fitsio.py

```diff
@@ -8,8 +8,13 @@
 @click.option('-d', '--directory', type=str, help="Specify path to directory of fitsfiles.", required=False)
 @sdi.generator
 def read(directory):
+    # add try (evaluate if the try is efficient)
     if directory == None:
-        paths = glob.glob("{}/*.fits*".format(tkinter.filedialog.askdirectory()))
+        try:
+            paths = glob.glob("{}/*.fits*".format(tkinter.filedialog.askdirectory()))
+        except:
+            print("Visual file dialog does not exist, please specify path to directory to save fitsfiles.")
```

Comment:
Use `click.echo` with `err=True` over `print` when possible!

Reply:
Also, this error message could be a little more specific: e.g., specify path to directory to _read_ fitsfiles with `-d` or something.